CONTRIBUTING.md

```diff
@@ -292,6 +292,7 @@ checklist.)
 2. Unit tests included and/or updated with changes?
 3. Command line build passes?
 4. Expect to pass code review?
+5. Changes have been smoke-tested?
```

Comment:
I also believe that code style should be enforced by an automated tool so we don't have to apply it ourselves. Wastes time in reviews.

Jslint currently fills the gap but I think we should eventually transition to jscs and eslint.  More configurable.

Pete

On Dec 2, 2015, at 11:14 AM, akhenry <notifications@github.com<mailto:notifications@github.com>> wrote:


In CONTRIBUTING.md<https://github.com/nasa/openmctweb/pull/349#discussion_r46460030>:

> @@ -292,6 +292,7 @@ checklist.)
>  2. Unit tests included and/or updated with changes?
>  3. Command line build passes?
>  4. Expect to pass code review?
> +5. Changes have been smoke-tested?


That sounds reasonable to me.

—
Reply to this email directly or view it on GitHub<https://github.com/nasa/openmctweb/pull/349/files#r46460030>.